### DISCUSSION TITLE
Show all files/sub-directories in current directory when prompt is blank

### DISCUSSION THREAD

**Discussion by Fubs** [marlonrichert/zsh-autocomplete#510] [Q&A] [answered]
The default behavior for a blank prompt is to show the (non-hidden) directories in the current directory, like in this screenshot:

![emptyCompletion](https://user-images.githubusercontent.com/17634481/201385695-81d593f8-98b8-4ba1-9f40-dce3561aa1f5.png)

I find this feature useful, but I also want a blank prompt to show the files inside the current directory, like the completions shown for the prompt "vim " : 

![vimCompletion](https://user-images.githubusercontent.com/17634481/201387173-03434bfd-a6fd-4597-85cd-02e0dc8d10c4.png)

Is there a config option to do that?

**Comment by marlonrichert** [comment, marked as answer]
Yes, there is. Add to your `.zshrc` file:
```zsh
setopt GLOB_DOTS
```
Read the [Zsh manual](https://zsh.sourceforge.io/Doc/Release/Options.html#index-GLOBDOTS) for more info.
